src/DetailPage.tsx

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink, useNavigate, useParams } from 'react-router-dom';
+
+export default function CourseDetail() {
+  const { id } = useParams(); // get the id of the course
+  const [title, setTitle] = useState('dummy title');
+  const [rating, setRating] = useState(4.7);
+  const [rateNum, setRateNum] = useState(19);
+  const [description, setDescription] = useState('dummy description');
+  const [points, setPoints] = useState([]);
+  const [destination, setDestination] = useState('dummy destination');
+  const [u_counts, setCounts] = useState(45);
+  const [e_time, setTime] = useState(50);
+  const [created_at, setCreateAt] = useState();
+  const [p_counts, setP] = useState(0);
```

Comment:
변수 이름 camelCase로 바꿔주세용

```suggestion
  const [expectedTime, setExpectedTime] = useState<number>(50);
  const [createdAt, setCreatedAt] = useState();
  const [passCounts, setPassCounts] = useState<number>(0);
```